msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/joined/TestCase832430.java

```diff
@@ -43,11 +44,10 @@
 
 import org.junit.Test;
 
-import java.util.concurrent.TimeUnit;
-
 // [Joined][MSAL] Acquire Token + Acquire Token Silent with resource (Prompt.SELECT_ACCOUNT)
 // https://identitydivision.visualstudio.com/DevEx/_workitems/edit/832430
 @RetryOnFailure(retryCount = 2)
+@DoNotRunOnPipeline("This seems broken with daily libraries, but not release? Getting some app permission error when signing in with truemam ca account")
```

Comment:
Can we investigate and fix it rather than ignoring it?

Reply:
This happened before the last release as well, most likely something to do with how the daily libs are built, I'll investigate but i don't think i'll have time to fully fix this before the coming release, hence why i'm ignoring them for this month

Reply:
I'll remove these annotations, we can track the failures on daily pipeline as expected and attempt a fix